.github/workflows/release.yaml

```diff
@@ -0,0 +1,372 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+permissions:
+  actions: write
+  contents: write
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  release:
```

Comment:
Is "release" all lowercase due to naming conflict?

Reply:
I don't know if there was a naming conflict, I had just made it lower case since I had seen examples using this action where the job names were lower case.